src/cli-options.ts

```diff
@@ -392,6 +409,13 @@ const cliOptions: CLIOption[] = [
       'Include only package names matching the given string, wildcard, glob, comma-or-space-delimited list, /regex/, or predicate function.',
     type: 'string | string[] | RegExp | RegExp[] | FilterFunction',
   },
+  {
+    long: 'filterMeta',
+    arg: 'fn',
+    description: `Filter on Packument using predicate function.`,
```

Comment:
As an average user, this description creates more questions than it answers. It'd be helpful if the description was more like the extended help docs.

Suggestion: `Filter packages given the metadata about the new version.`